andela/travel_tool_front
# #160715616 Check in to guesthouse

#### What does this PR do?
This provides an interface for a user to check in to a guesthouse

#### Description of Task to be completed?
As a requester, I should be prompted to check-in, on Travela, to confirm my arrival at the guest house centre
So that, travel team members at my destination and origin can be aware of my arrival at the centre

#### How should this be manually tested?
- Clone this repo - `git clone https://github.com/andela/travel_tool_front.git`
- Switch to the repo directory - `cd travel_tool_front`
- Check out this branch - `git checkout ft-checkin-guesthouse-160715616`
- Install dependencies - `yarn install`
- Start the server  - `make start`
- Set up the server by following this [guide](https://github.com/andela/travel_tool_back/pull/80)
- Navigate to `localhost:3000` in a browser
- Navigate to the booked rooms page using the link provided in the accommodation section on the left side navigation bar
- You should see your booked room record with a check-in button
- Click on the check-in button to check in to your guest house
- You should get a notification stating you just checked in to your guest house

#### Any background context you want to provide?
- You have to set up yourself as a manager so you can approve your request
- Ensure you follow the steps on the backend guideline so you have an existing approved request in order to check in successfully

#### What are the relevant pivotal tracker stories?
[#160715616](https://www.pivotaltracker.com/story/show/160715616)


#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-10-04 at 9 53 04 am" src="https://user-images.githubusercontent.com/13545380/46463998-38c54980-c7bd-11e8-8254-08c87f0e245f.png">
<img width="1440" alt="screen shot 2018-10-04 at 9 57 51 am" src="https://user-images.githubusercontent.com/13545380/46464000-38c54980-c7bd-11e8-9149-7a89cd6476bc.png">
<img width="1440" alt="screen shot 2018-10-04 at 9 58 02 am" src="https://user-images.githubusercontent.com/13545380/46464001-395de000-c7bd-11e8-83cd-5a6065ecbc45.png">
<img width="1440" alt="screen shot 2018-10-04 at 9 58 29 am" src="https://user-images.githubusercontent.com/13545380/46464002-395de000-c7bd-11e8-820e-34344331a806.png">




#### Questions:
